ipc/src/main/java/com/c8y/ms/templates/ipc/service/TemplatesBasicClient.java

```diff
@@ -0,0 +1,14 @@
+package com.c8y.ms.templates.ipc.service;
+
+import java.util.List;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@FeignClient(name = "basic", url = "https://localhost:8080/service/templates-basic", configuration = TemplatesBasicClientConfig.class)
```

Comment:
will this work when deployed to C8Y?

Reply:
yes, unfortunatly the url is mandatory, the request interceptor will overwrite the URL anyways.

Reply:
oh sorry, isn't mandatory, i will remove the url parmater

Reply:
@PestusAtSAG it seems the url param is still there ...

Reply:
yes, i had to add  it again, wasn't working after i removed it.